mod-orders-storage/schemas/purchase_order.json

```diff
@@ -78,6 +78,14 @@
       "type": "string",
       "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
     },
+    "units": {
```

Comment:
No need to add units to storage because only composite order should be aware about it.